docs/index.md

```diff
@@ -34,6 +34,7 @@ License <misc/license_link>
 
 User Guide <userguide/userguide>
 Examples <examples/examples>
+Versions <https://avhopp.github.io/baybe_dev/versions>
```

Comment:
hardlink to your fork, intended?

Reply:
This will be changed once you are happy with the look and feel - this was necessary for testing.

Reply:
i ams till confused between you opening PRs for testing/ reviewing. Which is this one for? look and feel are fine to me, suggestions about that included in other comments

Reply:
This PR is the final one, with the only exception being the links that still need to be fixed. My goal was to demonstrate *exactly* what the result looks like and to provide the *exact* code that produced this on my source, because as far as I understood it, you guys wanted to see exactly that.

Reply:
I will work on all other things and change the links at the very end - otherwise you won't be able to actually see the difference.

Reply:
Changed.